src/ServiceManagement/StorSimple/Commands.StorSimple/Exceptions/StorSimpleDeviceNotFoundException.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.WindowsAzure.Commands.StorSimple.Exceptions
+{
+    [Serializable]
+    public class StorSimpleDeviceNotFoundException : Exception
```

Comment:
What is the point of having a class just to use generic error message? Could you just have a method in base class or something that's called `GetGenericException` which will return `new Exception(""The device name provided does not exist under your currently selected resource. To get a list of all available devices use Get-AzureStorSimpleDevice"")`?

Btw the code line is more than 100 chars which make it hard to review